lib/presentation/home/widgets/weather_days_list.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:db_course_app/resources/images.dart';
+import 'package:flutter/material.dart';
+
+const TextStyle textStyle = TextStyle(
```

Comment:
Extract text styles to resources. Omit the type on defining vars 

`const TextStyle textStyle = TextStyle(` =>

`const textStyle = TextStyle(`